content/authoring/guidelines/sample-tests.md

```diff
@@ -7,4 +7,23 @@ next: /authoring/guidelines/submission-tests/
 
 # Writing Sample Tests
 
-_This page is a stub._
+Sample tests are a part of every kata which is often very underappreciated by both kata authors, and users attempting it. Bad sample tests can cause a lot of frustration, while a good suite of them often increases overall satisfaction of users from solving the kata. To make sample tests as good as possible, follow guidelines collected below.
+
+- **Conform to [General Coding Guidelines][authoring-guidelines-general-coding]**: tests are code too and as such should keep up to code quality standards.
```

Comment:
```suggestion
- **Conform to [General Coding Guidelines](/authoring/guidelines/coding/)**: tests are code too and as such should keep up to code quality standards.
```

or add `[authoring-guidelines-general-coding]` at the bottom of the file.

```markdown
[authoring-guidelines-general-coding]: /authoring/guidelines/coding/
```

I prefer the latter because it's easier to maintain and read raw Markdown.

Reply:
I remembered to add the reference at the bottom, and then forgot 🤦 

Added now.